ci/ci/build.py

```diff
@@ -190,7 +184,7 @@ def __init__(self, params):
         self.clouds = json.get('clouds')
         self.run_if_requested = json.get('runIfRequested', False)
 
-        self.token = generate_token()
+        self.token = params.token
```

Comment:
Hmm. I *think* this is OK, but why do this? Doesn't seem bad for each step to get separate randomness.